vspace/src/main/webapp/WEB-INF/views/staff/modules/module.jsp

```diff
@@ -125,8 +176,27 @@ $(document).ready(function($) {
 						</div>
 					</div>
 				</c:forEach>
-			</div>
+			</div>	
 			<div id="selectedSequence" class="col justify-content-center" style="padding-left: 60px; padding-right:20px;"></div>
 		</div>
 	</div>
 </div>
+
+  
+<div id="deleteSlideAlert" class="modal fade" role="dialog" >
+	<div class="modal-dialog">
+	<div class="modal-content">
+	<div class="modal-header">
+		<h4 class="modal-title" id="deleteModalTitle">Confirm Deletion?</h4>
+		<button type="button" id="closeSlide" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+	</div>
+	<div class="modal-body">
+		<p> This Slide is a part of another Sequence. Are you sure you want to delete it?</p>
```

Comment:
This should say:
"This slide is part of X sequence(s). Are you sure you want to delete it?"

where X is the number of sequences it's part of. Even better, it should list the sequences it's part of.